README.md

```diff
@@ -44,6 +44,21 @@ See Helm `values.yaml`, `enterprise` section for configuration examples.
 > Don't have StackStorm Enterprise License?<br>
 > 90-day free trial can be requested at https://stackstorm.com/#product
 
+## Configuration
+
+The default configuration values for this chart are described in `values.yaml`.
+
+## Ingress
```

Comment:
Just a reminder that after merging this PR we'll need to sync the respective `st2docs` for K8s/HA